cookbooks/ruby/attributes/default.rb

```diff
@@ -1,6 +1,8 @@
-env_var_ruby = fetch_env_var(node, "EY_RUBY_VERSION")
-if env_var_ruby
-  default[:ruby][:version] = env_var_ruby
-else
-  default[:ruby][:version] = node.engineyard.environment.ruby[:version]
+if node.engineyard.environment.ruby?
```

Comment:
Is this `if` statement required? I see the same functionality here: https://github.com/engineyard/ey-cookbooks-stable-v6/blob/next-release/cookbooks/ey-base/recipes/bootstrap.rb#L108

Reply:
Yes it's required. `node.engineyard.environment.ruby[:version]` fails on a php environment. And since this code is in an attributes file, it's called even if the `ruby` recipe is not included.

`ey-base::bootstrap` needs `if node.engineyard.environment.ruby?` because this recipe is called on all environments including php.